Formula/ffmpeg.rb

```diff
@@ -146,6 +147,12 @@ def install
     args << "--enable-openssl" if build.with? "openssl"
 
     # packages that need additional license options
```

Comment:
Actually, that comment is a little misplaced. We should group license-related code and build-related code.

Reply:
indeed! updated